beta-pipelines/multiome/atac.wdl

```diff
@@ -299,31 +310,49 @@ task CreateFragmentFile {
     python3 <<CODE
 
 
+    atac_gtf = "~{atac_gtf}"
     barcodes_in_read_name = "~{barcodes_in_read_name}"
     bam = "~{bam}"
     bam_base_name = "~{bam_base_name}"
+    chrom_sizes = "~{chrom_sizes}"
+    
+    # Calculate chrom size dictionary based on text file
+    chrom_size_dict={}
+    with open('~{chrom_sizes}', 'r') as f:
+        for line in f:
+            key, value = line.strip().split()
+            chrom_size_dict[str(key)] = int(value)
 
     # if barcodes are in the read name, then use barcode_regex to extract them. otherwise, use barcode_tag
 
     if barcodes_in_read_name=="true":
       import snapatac2.preprocessing as pp
```

Comment:
`import snapatac2.preprocessing as pp`

`import snapatac2 as snap`

Both these imports are used by both conditions in the `if statement`. Thus, I think it can be placed once before the `if statement`.

Reply:
thank you! just made those updates